src/components/pages/ParentHome/ParentResources.js

```diff
@@ -1,33 +1,42 @@
 import React from 'react';
+import Banner from '../../common/Banner';
+import ParentSidebar from '../ParentHome/ParentSidebar';
+import ParentNavbar from './ParentNavbar';
+import StudentDropdown from './StudentDropdown';
+import '../../../styles/ParentStyles/index.less';
+import { Layout, Card, Empty } from 'antd';
 import 'antd/dist/antd.css';
-import { Space, Card } from 'antd';
-import { MailOutlined } from '@ant-design/icons';
 
-const ParentResources = props => {
+const ParentResourcesPage = () => {
+  const { Content } = Layout;
   return (
-    <div>
-      <h2 style={{ fontSize: '2.5rem', textAlign: 'center' }}>
```

Comment:
I like that you removed the styling from the html here and kept it in a separate less file